gui/include/gui/printout_chart.h

```diff
@@ -23,29 +18,32 @@
  *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,  USA.         *
  **************************************************************************/
 
-#ifndef _OCPN_PRINT_H__
-#define _OCPN_PRINT_H__
+#ifndef PRINTOUT_CHART_H
+#define PRINTOUT_CHART_H
 
-#include <wx/dc.h>
-#include <wx/bitmap.h>
-#include <wx/print.h>
-#include <wx/string.h>
+#include "printout_base.h"
 
-class MyPrintout : public wxPrintout {
+class ChartPrintout : public BasePrintout {
 public:
-  MyPrintout(const wxChar *title = _T("My printout")) : wxPrintout(title) {}
-  virtual bool OnPrintPage(int page);
-  virtual bool HasPage(int page);
-  virtual bool OnBeginDocument(int startPage, int endPage);
-  virtual void GetPageInfo(int *minPage, int *maxPage, int *selPageFrom,
-                           int *selPageTo);
+  ChartPrintout(const wxString &title);
 
-  void DrawPageOne(wxDC *dc);
+  /**
+   * Print chart.
+   * @param page Page number.
+   */
+  bool OnPrintPage(int page) override;
```

Comment:
Probably better to just drop the documentation here. Doxygen will then use the  inherited wxw docs whioch are better. This is a general  remark  for all overrides.